### ISSUE TITLE
Will this package keep update?

### ISSUE THREAD

**Issue by Creolophus** [fireslime/photo_view#355]
I mean this is a fantastic photo view package for me. But I found it is no new release since Aug.20.
And for me there is a gesture issue on Android since flutter 1.20.

**Comment by agordeev** [comment]
I used this library too, but switched to native solution. It's quite easy to achieve the same result with `PageView` and the new `InteactiveViewer` widget.

**Comment by renancaraujo** [comment]
I'm really sorry for that amount of time without any release. 
I've created this two years ago not having the idea of the amoutn of people tha would end up using it. Its really great to know that something you have done prectically alone helps so many around the world. 
But unfortunately I got caught pretty hard by the 2020 events and its being really hard to manage time to keep this up to date. 

`InteactiveViewer` is great and I recomend you to use it if you dont necessarily need the features that only PhotoView has.

I do have big plans for this inclusing a two axis scrolable, scroll physics and possibly extending interactive viewer, but as it is right now (being all alone maintinning this) I dont think I will have the necessary pace to even keep up to date with the isssues.
